src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -213,17 +213,9 @@ The .NET Foundation licenses this file to you under the MIT license.
     <MSBuild Projects="@(ProjectToBuild)" BuildInParallel="true" />
   </Target>
 
-<<<<<<< HEAD
-  <!-- NativeAOT-LLVM note: 'BeforeTargets="_RunILLink"' should be deleted once https://github.com/dotnet/runtime/pull/63914 is merged from upstream -->
-  <Target Name="IlcCompile"
-      Inputs="@(IlcCompileInput);@(RdXmlFile)"
-      Outputs="$(NativeIntermediateOutputPath)%(ManagedBinary.Filename)$(IlcOutputFileExt)"
-      BeforeTargets="_RunILLink"
```

Comment:
We may need to bring this `BeforeTargets="_RunILLink"` back and reword the `TODO` to say `once upstream uses .NET 7 SDK`.